releases/25.10.0/25.10.0.md

```diff
@@ -367,3 +367,4 @@ The `WarehouseAutomation` Gem is planned to be removed in the next release as it
     ```sh
     cmake -B build/linux -S . -G "Ninja Multi-Config" -DCMAKE_C_COMPILER=clang-18 -DCMAKE_CXX_COMPILER=clang++-18
     ```
+- O3DE 25.10 requires that Git must be installed on the system to build projects. You can try to install from https://git-scm.com/install/windows in case a CMake error occurs while building. The requirement will be removed in the next 25.10.1 version.
```

Comment:
I suggest wording it like this:

O3DE 25.10 requires that Git be installed in order to fetch and build projects. A Windows based installer can be located at https://git-scm.com/install/windows in the event that a CMake error occurs while building a project. The requirement will be removed in the next point release v25.10.1.